### PR TITLE
Docs: Clean up API documentation (#503)

### PR DESCRIPTION
**Pull request by wll8**
### Use docsify to create the document.
- `npm i docsify-cli -g`
- `docsify serve docs`

#### Checklist
* [ ] New tests have been added
* [ ] `npm test` passes successfully
* [x] Documentation has been updated

@amir20 to review

### CODE REVIEW

File: docs/zh-cn/_navbar.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- Translations
+  - [:uk: English](../)
+  - [:cn: 中文](../zh-cn/)
```
Comment (amir20) [inline docs/zh-cn/_navbar.md]: Wow thank you for this

**Comment by amir20** [comment]
How should this be done so that people can click the README and go to DOCS?

**Comment by wll8** [comment]
@amir20 
You can deploy documentation to 'GitHub Pages' or your own website.

In the warehouse: https://github.com/wll8/phantomjs-node/tree/master/docs

You can even switch languages directly in the repository: https://github.com/wll8/phantomjs-node/blob/master/docs/_navbar.md

Deploy elsewhere: https://docsify.js.org/#/deploy?id=deploy.

**Comment by amir20** [comment]
This is live now on http://amirraminfar.com/phantomjs-node/#/ 🙌 thank you!!!

**Pull request merged**